boot-qemu.sh

```diff
@@ -203,6 +203,12 @@ function setup_qemu_args() {
             QEMU=(qemu-system-riscv64)
             ;;
 
+        s390)
+            KIMAGE=bzImage
+            QEMU_ARCH_ARGS=(-M s390-ccw-virtio)
+            QEMU=(qemu-system-s390x)
```

Comment:
Using "-smp", as done on x86-64 AFAIKs might reduce boot times further. SMP under QEMU/TCG with MTTCG (one thread per vCPU, usually the default) has been working reliably for some years now.